### PR TITLE
Localize to Polish

### PR DESCRIPTION
**Pull request by lwojcik**
[You called](https://veganism.social/@vegancheck/109915088244873553) and [I responded](https://masto.ai/@lukem/110264134377295841) 😃 🇵🇱 

Additional notes:

- I didn't translate TOS as I didn't see any existing translations other than English and German. Your project readme doesn't state that but I assume that's intentional as it's a legal document?

- The font used in the project renders Polish diacritic characters in a very ugly way but I left this out of scope of this PR - this problem requires additional investigation. I suspect updating webfont files to support latin-extended Unicode block may fix the problem.

- in Polish, unlike in English, we use [declension](https://en.wikipedia.org/wiki/Declension) and current localization setup makes it a bit tricky to work with. For example, with current setup I translate the phrase:

> Made with  by [Philip Brembeck](https://philipbrembeck.com/) & [FrontEndNet.work](https://frontendnet.work/)

as

> Zbudowane z ❤️ przez [Philip Brembeck](https://philipbrembeck.com/) i [FrontEndNet.work](https://frontendnet.work/)

but that doesn't sound natural as I can't modify the author's name in any way. The natural version should look like this:

> Zbudowane z ❤️ przez [Philipa Brembecka](https://philipbrembeck.com/) i [FrontEndNet.work](https://frontendnet.work/)

Same applies to sharing feature. Currently I translate

> Share on Mastodon

as

> Udostępnij na Mastodon

but it doesn't great. The correct translation should be:

> Udostępnij na Mastodonie

The alternative could also be:

> Udostępnij w sieci Mastodon

which literally means 'Share on Mastodon network', is formally correct but way too wordy.

In some cases it's possible to work around this limitation with clever use of words and grammar, but believe me, as a Polish native I can spot those shenanigans as they happen. 😸 

Please let me know if that helps. Thanks!

### CODE REVIEW

**Comment by philipbrembeck** [comment]
Wow, thank you a lot! I'm gonna review the changes you did and open up issues for the issues you pointed out. 
If you want to, I will keep you updated and maybe after the implementation of fixes for those issues, you could update the polish files! 😊 

Thanks again! 💯

File: src/locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  "Ingredients": {
+    "ingredientcheck": "Weryfikator Składników Wegańskich",
+    "ingredientcheck_desc": "Sprawdź, czy składniki produktu są wegańskie! Wprowadź składniki oddzielając je przecinkiem",
+    "entercommaseperated": "Wprowadź składniki oddzielone przecinkiem",
+    "submit": "Wyślij",
+    "vegan": "Wegańskie",
+    "source": "Źródło danych",
+    "cannotbeempty": "Lista składników nie może być pusta, zawierać znaków specjalnych i musi być oddzielona przecinkiem!",
+    "licenses": "Licencje",
+    "licenses_desc": "VeganCheck.me uses different databases and APIs to gather information about a product. The information is licensed under the following licenses:",
```
Comment (philipbrembeck) [inline src/locales/pl.json:98]: You're right, there's no need to translate this, as it's only legal stuff. Good one!

File: src/locales/pl.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
... our services against payment is strictly prohibited. Automated queries are also prohibited and may only be performed using the <a href=\"https://jokenetwork.de/vegancheck-api\">API</a>.<br> We reserve the right to change, reduce, expand or disable our services at any time.</p><p><b>Processing of entered texts and numbers</b></p><p>On our website, barcodes or ingredients of a product can be entered in text fields. This is necessary to provide our service in its entirety. The data entry is completely voluntary.<br>Only barcodes or ingredients may be entered in the named text fields; the entry or transmission of personal data of any kind is strictly prohibited.</p><p>Personal data is any information that relates to an identified or identifiable living individual. Different pieces of information, which collected together can lead to the identification of a particular person, also constitute personal data.<sup><a href=\"https://ec.europa.eu/info/law/law-topic/data-protection/reform/what-personal-data_en\">?</a></sup></p><p><b>Disclaimer</b></p><p>We assume no liability for the accuracy of the product information and ingredient lists offered and for the availability of our service.</p>"
```
Comment (philipbrembeck) [inline src/locales/pl.json:62]: Agreed, this does not have to be translated 😊

**Review by philipbrembeck — APPROVED** [review]

**Comment by lwojcik** [comment]
> I will keep you updated and maybe after the implementation of fixes for those issues, you could update the polish files! 😊

Sure! If you add any new phrases or modify the system in any way I'll be happy to help.
